sc_rf_service/sc_rf_service.py

```diff
@@ -382,25 +372,45 @@ def __init__(self):
         self.cmd_socket.connect("tcp://127.0.0.1:{}".format(os.environ.get('MODEL_PORT', 12312)))
         init_vals = self.get_cavity_ACTs_from_model()
 
-        for d in init_vals.keys():
-            print(d)
-            print(init_vals[d])
-            print(type(init_vals[d]))
+        #print(init_vals)  # i.e. {..., 'ACCL:L3B:3450': (14454000.0, 0.0, 621.36, 'CAVL345', 1.038, 'T')}
+
+
+
         cav_pvs = {device_name: CavityPV(device_name, self.on_cavity_change, initial_values=init_vals[device_name],
                                          prefix=device_name) for device_name in init_vals.keys()}
-
+        #print(init_vals["ACCL:L3B:3450"])
+        #print(init_vals)
         # cav_pvs = {device_name: CavityPV(device_name, self.on_cavity_change, ppp=init_vals[device_name][0], prefix=device_name) for device_name in init_vals.keys()}
 
         # setting up convenient linac section PVs for changing all of the L1B/L2B/L3B cavities simultaneously.
-        print(cav_pvs.keys())
+        #print(cav_pvs.keys())
         linac_init_vals = _make_linac_table(init_vals)
-        print(linac_init_vals)
+        #print(linac_init_vals)
         # linac_pvs = {device_name: CavityPV(device_name, self.on_cavity_change, initial_values=linac_init_vals[device_name], prefix=device_name) for device_name in linac_init_vals.keys()}
 
         self.add_pvs(cav_pvs);
         # self.add_pvs(linac_pvs);
         L.info("Initialization complete.")
 
+        #self.get_rackPV_prefix()
+
+    def get_rackPV_prefix(self):
+        DISPLAY_LINAC_OBJECTS: List[Linac] = []
+
+        for name, cryomoduleList in LINAC_TUPLES:
+            displayLinac = Linac(name, cryomoduleList)
+            DISPLAY_LINAC_OBJECTS.append(displayLinac)
```

Comment:
Is there a reason you're recreating this instead of importing the one from scLinac?

Reply:
No particular reason; I just copied and pasted. Thanks for pointing it out:D

Reply:
Yeah I'd change it to import!

Reply:
Fixed!